### DISCUSSION TITLE
I can't create public environment with my username

### DISCUSSION THREAD

**Discussion by Amshaker** [community/community#56069] [Repositories]
### Select Topic Area

Bug

### Body

I want to create a public environment with my username. But I can't due to this error:
The repository Amshaker has been retired and cannot be reused.

**Comment by mrepol742** [comment]
Try changing your accnt username to different one and create a repository with the name Amshaker then revert your username back if it didnt work i guess you should change your repo name.

according to https://github.blog/2018-04-18-new-tools-for-open-source-maintainers/#popular-repository-namespace-retirement
Many package managers allow developers to identify packages by the maintainer’s login and the project name, for example: Microsoft/TypeScript or swagger-api/swagger-codegen. This is an efficient way to describe a dependency, but sometimes maintainers delete or rename their accounts, allowing developers to intentionally or unknowingly create projects with the same name.

To prevent developers from pulling down potentially unsafe packages, we now retire the namespace of any open source project that had more than 100 clones in the week leading up to the owner’s account being renamed or deleted. Developers will still be able to sign up using the login of renamed or deleted accounts, but they will not be able to create repositories with the names of retired namespaces.
